modules/core/src/main/java/org/apache/ignite/internal/processors/task/monitor/ComputeTaskStatus.java

```diff
@@ -64,6 +64,9 @@
     /** Reason for the failure of the task. */
     @Nullable private final Throwable failReason;
 
+    /** Availability of changing task attributes. */
+    private final boolean fullSupported;
```

Comment:
How about renaming the field and all corresponding accessor methods to a simpler `fullSupport`? In that case it matches with `GridTaskSessionInternal#isFullSupport`.

Reply:
Yes, I think it can be renamed.